src/file.cpp

```diff
@@ -67,7 +67,7 @@ POSSIBILITY OF SUCH DAMAGE.
 #include "libtorrent/aux_/disable_warnings_pop.hpp"
 
 #include "libtorrent/config.hpp"
-#include "libtorrent/alloca.hpp"
+#include "libtorrent/aux_/alloca.hpp"
```

Comment:
it appears file.cpp is not missing ``aux::numeric_cast`` for some reason. not obvious why that happened